content/javascript/concepts/optional-chaining/optional-chaining.md

```diff
@@ -0,0 +1,167 @@
+---
+Title: 'Optional Chaining'
+Description: 'The optional chanining operator simplifies the comparison of multiple data properties in a chain of connected objects' 
+Subjects: 
+  - 'Web Development'
+  - 'Computer Science'
+Tags: 
+  - 'Data'
+  - 'Operators'
+CatalogContent: 
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+
+
+The **optional chanining** operator simplifies the comparison of multiple data properties in a chain of connected objects. This is especially valuable if any of the properties are null as the operatior will return as undefined instead of throwing an error.
+
+Optional chaning can be used to validate nested object properties, call optional functions or methods on optional objects, accessing arrary elements, and more. 
+
+The primary benefits of using optional chaining instead of regular syntax include simplifying the code to achieve the same result so it's easier to read, protecting against runtime errors and enhances maintainability.
```

Comment:
"Enhances" can be changed to "enhancing" to maintain a parallel structure, and we should add a comma for clarity.

```suggestion
The primary benefits of using optional chaining instead of regular syntax include simplifying the code to achieve the same result, making it easier to read, protecting against runtime errors, and enhancing maintainability.
```